db/migrations/20180910233720_create_deal.sql

```diff
@@ -17,12 +18,9 @@ CREATE INDEX deal_bid_id_index
     ON maker.deal (bid_id);
 CREATE INDEX deal_address_index
     ON maker.deal (address_id);
+CREATE INDEX deal_msg_sender_index
+    ON maker.deal (msg_sender);
 
 
 -- +goose Down
-DROP INDEX maker.deal_address_index;
```

Comment:
Did you mean to remove this drop? I guess maybe it's unnecessary since you're getting rid of the columns.

Reply:
Yeah just figured removing it entirely was probably more simple than adding another one for the new index. Had previously been a fan of being explicit about the drops, but it seems unnecessary if we lose them when we drop the table anyway (and makes it all to easy to forget one).